oslo_cache/backends/redis.py

```diff
@@ -0,0 +1,45 @@
+from dogpile.cache.backends import redis as redis_backend
+from oslo_cache import core
+__all__ = [
+    'RedisSentinelTLSBackend'
+]
+class RedisSentinelTLSBackend(redis_backend.RedisSentinelBackend):
```

Comment:
```suggestion
class RedisSentinelBackend(redis_backend.RedisSentinelBackend):
```

As the class supports TLS and non-TLS, I would probably drop the TLS from the name.